.openpublishing.redirection.json

```diff
@@ -2728,6 +2728,11 @@
       "redirect_url": "/dotnet/desktop-wpf/fundamentals/xaml",
       "redirect_document_id": true
     },
+    {
+      "source_path": "docs/framework/wpf/app-development/how-to-determine-the-installed-version-of-wpf.md",
+      "redirect_url": "https://docs.microsoft.com/previous-versions/dotnet/netframework-3.5/aa349641(v=vs.90)",
+      "redirect_document_id": true
```

Comment:
I'm not sure what the `redirect_document_id` does. I just copied from the previous redirection.

Reply:
It should be fine to set it like you have. It's used for tracking page views, etc.

Reply:
True doesn’t work for articles that are not on the same repo

Reply:
@mairaw, Ok. Then I'm going to remove it.